src/org/omegat/util/gui/UIDesignManager.java

```diff
@@ -69,16 +72,84 @@
  * @author Alex Buloichik
  * @author Yu Tang
  * @author Aaron Madlon-Kay
+ * @author Hiroshi Miura
  */
 public final class UIDesignManager {
 
+    public static final String THEME_DEFAULT_ICON = "thumb_default.png";
+    public static final String THEME_UNKNOWN_ICON = "thumb_unknown.png";
+
+    private static final Map<String, URL> themeImages = new HashMap<>();
+    private static final Map<String, IThemeDesigner> themeInitilizer = new HashMap<>();
```

Comment:
These are `static` and `final` so they should be UPPER_CASE.

Also there is a typo: `themeInitilizer` → `THEME_INITIALIZER`